_includes/sidebar.html

```diff
@@ -43,5 +43,6 @@
     </p>
 
     <p>&copy; {{ site.time | date: '%Y' }} <a href="https://github.com/doomspork">Sean Callan</a></p>
+    <p><a href="/sitemap.xml">sitemap</a></p>
```

Comment:
This link probably isn't necessary now that the robots.txt links to the sitemap.

I'll :scissors:  it